index.js

```diff
@@ -30,17 +30,24 @@ const getState = (key) => {
   return state[key]; // WARNING: mutable
 };
 
+const _prompt = async (prompt, customConf, message) => {
+  const promptObj = {
+    ...prompt,
+    ...customConf
+  };
+  const answer = await inquirer.prompt([{ ...promptObj, prefix: meta.prefix, message }]);
+  return answer.value;
+};
+
 // reusable prompt; wraps inquirer
 // e.g. const name = await prompot('What is your name?');
-const prompt = async (message) => {
-  const answer = await inquirer.prompt([{ ...meta.prompt, prefix: meta.prefix, message }]);
-  return answer.value;
+const prompt = async (message, customConf = {}) => {
```

Comment:
can you rename `customConf` to `options`? that's the usual name.

Reply:
Done